src/main/haskell/language-kore/src/Data/Kore/AST/Common.hs

```diff
@@ -101,14 +109,16 @@ data Id level = Id
     { getId      :: !String
     , idLocation :: !AstLocation
     }
-    deriving Show
+    deriving (Show, Generic)
```

Comment:
Just curious, not asking for a fix: Where is this generic thing used (for any of the instances defined here, not necessarily this one)?

Reply:
Generic is needed to derive classes like Hashable.